collections/_article/dev-snapshot-godot-4-4-dev-6.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "Dev snapshot: Godot 4.4 dev 6"
+excerpt: "Featuring: Malware! (does not contain malware)"
+categories: ["pre-release"]
+author: Thaddeus Crews
+image: /storage/blog/covers/dev-snapshot-godot-4-4-dev-6.webp
+image_caption_title: "Malware"
+image_caption_description: "A game by OddGames"
+date: 2024-12-04 12:00:00
+---
+
+Safety goggles on, people! Features and Quality-of-Life are being integrated at blinding speeds; precautions must be taken to view them head-on. Once secure, we can dive into what's shaping up to be our final development snapshot before 4.4-beta builds roll out!
+
+Many of the changes in this release are bug fixes that will be backported to Godot 4.3 and released in 4.3.1! So please
+test this release well so we can be confident with the changes and release 4.3.1 with them as soon as possible.
```

Comment:
I think we can drop this by now, it was mostly meant for the first dev snapshots as usually we release the first patch release around the same time as dev1 or dev2 with similar (bugfix) changes.
```suggestion
```